simple-blog/src/app/app.component.html

```diff
@@ -1,11 +1,14 @@
 <body>
   <!-- mat-toolbar needs to be refactored to be its own component so it can be reusable -->
   <mat-toolbar id="navbar" color="primary">
-    <div id="nav-title">Jose's Blog</div>
+    <div id="nav-title">
+      <a href="http://localhost:4200/">Jose's Blog</a>
+    </div>
     <div id="nav-content">
-      <a href="">Home</a>
-      <a href="">Login</a>
-      <a href="">Profile</a>
+      <button type="button">Home</button>
```

Comment:
I forgot to mention to remove the home button. It is unnecessary, now that the Simple-Blog title should take you back to the home page.